src/Simulation/Simulators/ResourcesEstimator/Simulator.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using Microsoft.Quantum.Simulation.Simulators;
+using Microsoft.Quantum.Simulation.Core;
+using Microsoft.Quantum.Simulation.QCTraceSimulatorRuntime;
+using Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators;
+
+// using System.IO;
+// using System.Threading.Tasks;
+
+namespace Simulator
+{
+    public class AdvancedSimulator : ResourcesEstimatorWithAdditionalPrimitiveOperations
```

Comment:
This class and its logic should be merged with the existing `ResourcesEstimator` class.